dist/notice/ds4/notice.css

```diff
@@ -56,28 +56,17 @@ button.page-notice__close span {
   display: inline-block;
   vertical-align: middle;
 }
-.flyout-notice,
```

Comment:
@seangates @prabhurangan Removing these `.flyout-notice` classes is a breaking change to the Notice module. I don't understand why they would be moved to the Tooltip module either.

Reply:
Oh, never mind, I see all those rules were moved within a media query.